internal/cmd/logout.go

```diff
@@ -36,20 +81,94 @@ func logout(purge bool) error {
 		return err
 	}
 
-	for i, hostConfig := range config.Hosts {
-		config.Hosts[i].AccessKey = ""
-
-		client, err := apiClient(hostConfig.Host, hostConfig.AccessKey, hostConfig.SkipTLSVerify)
-		if err != nil {
-			logging.S.Warn(err.Error())
-			continue
+	switch {
+	case all:
+		stateChanged := false
+		for i := range config.Hosts {
+			if config.Hosts[i].isLoggedIn() {
+				logoutOfServer(&config.Hosts[i])
+				stateChanged = true
+			}
+		}
+		if stateChanged {
+			fmt.Fprintf(os.Stderr, "  Logged out of all servers.\n")
+		} else {
+			fmt.Fprintf(os.Stderr, "  Not logged in to any servers.\n")
 		}
 
-		_ = client.Logout()
-	}
+		if clear {
+			if config.Hosts != nil {
+				config.Hosts = nil
+				fmt.Fprintf(os.Stderr, "  Cleared list of all servers\n")
```

Comment:
More of a suggestion, but we don't need to let the user know this as they specified it with `--clear` and it may be too much information